src/main/java/org/zalando/nakadi/exceptions/runtime/AccessDeniedException.java

```diff
@@ -26,6 +35,7 @@ public Resource getResource() {
     }
 
     public String explain() {
-        return "Access on " + operation + " " + resource.getType() + ":" + resource.getName() + " denied";
+        return "Access on " + operation + " " + resource.getType() + ":" + resource.getName() + " denied"
```

Comment:
Maybe we can improve the message further by checking if `null` == operation and show a different message in case it is. Something generic (like `Unable to complete operation on "event-type-name" + reason`. What do you think?

Reply:
Good point